consensus/qbft/src/test/java/org/hyperledger/besu/consensus/qbft/messagewrappers/ProposalTest.java

```diff
@@ -41,16 +42,15 @@
 import org.junit.Test;
 
 public class ProposalTest {
+  private static final BftExtraDataCodec bftExtraDataCodec = new QbftExtraDataCodec();
```

Comment:
Do we need a round trip test with CMS encoding? Might not be of much benefit as really just testing extraDataCodec?

Reply:
If we want a round trip test for the Proposal, it probably makes more sense to do it in the follow-up PR will all the PKI extraData CMS changes (the new format, etc.).